cmd/sealer/cmd/cluster/run.go

```diff
@@ -137,7 +157,7 @@ func NewRunCmd() *cobra.Command {
 	runCmd.Flags().StringSliceVar(&runFlags.CMDArgs, "cmd-args", []string{}, "set args for image cmd instruction")
 	runCmd.Flags().StringSliceVarP(&runFlags.CustomEnv, "env", "e", []string{}, "set custom environment variables")
 	runCmd.Flags().StringVarP(&clusterFile, "Clusterfile", "f", "", "Clusterfile path to run a Kubernetes cluster")
-
+	runCmd.Flags().StringSliceVar(&runFlags.LaunchCmds, "cmds", []string{}, "override default LaunchCmds of clusterimage")
```

Comment:
what will happen if I pass this flag for non-app image

Reply:
> 

I added the error message prompt
![image](https://user-images.githubusercontent.com/65949637/206982174-ae8ec240-f0dc-4281-8d2a-69fa6a979c69.png)